cli/command/cluster/create.go

```diff
@@ -21,33 +24,76 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.IntVarP(&opts.workers, "workers", "w", 2, "Initial number of worker nodes")
-	flags.IntVarP(&opts.managers, "managers", "m", 3, "Intial number of manager nodes")
+	flags.IntVarP(&opts.managers, "managers", "m", 3, "Initial number of manager nodes")
```

Comment:
the --managers and --workers are global options of the CLI, but the AWS plugin has its own options for node counts (depending on the template).
We should guess a default parameter name (the one from the Docker for AWS template is probably the best we can choose), and when apply the -m and -w values to these --aws-PARAMETER.